components/benefit_sponsors/app/domain/benefit_sponsors/operations/benefit_applications/reinstate.rb

```diff
@@ -127,6 +125,26 @@ def reinstate_census_employees
           Success()
         end
 
+        def handle_coverage(hbx_enrollment, reinstate_enrollment)
+          if hbx_enrollment.waiver_reason.present?
+            reinstate_enrollment.waive_coverage!
+          else
+            reinstate_enrollment.begin_coverage!({ disable_callbacks: true })
```

Comment:
`reinstate_enrollment.begin_coverage!({ disable_callbacks: true })`is  duplicated code, could you please remove it?

Reply:
` 132  reinstate_enrollment.begin_coverage!({ disable_callbacks: true })
          133 reinstate_enrollment.begin_coverage!({ disable_callbacks: true }) if TimeKeeper.date_of_record >= reinstate_enrollment.effective_on && reinstate_enrollment.may_begin_coverage?`
          If you see line 132 and 133 it was written without condition and with condition

Reply:
right; as i already said earlier - 

at 132 - it moves enrollment from coverage_reinstated to coverage_selected.
at 133 - (this only triggers when enrollment is already effective) - it moves enrollment from coverage_selected to coverage_enrolled.

Reply:
thanks for explanation